WSFrontend/org.openl.rules.ruleservice.ws/src/org/openl/rules/ruleservice/oauth/JWTValidator.java

```diff
@@ -53,13 +63,9 @@ public void setJwkReaders(List<JWKTokenReader> jwkReaders) {
      * @throws ParseException If the string couldn't be parsed to a valid signed JWT.
      * @throws BadJWTException If validation fails for some other reason.
      */
-    public void validateToken(String jwtToken) throws ParseException, JOSEException, BadJWTException {
+    private void validateToken(String jwtToken) throws ParseException, JOSEException, BadJWTException {
         SignedJWT jwsToken = SignedJWT.parse(jwtToken.replaceFirst("Bearer", ""));
```

Comment:
Extract `Bearer ` to a constant. Use `substring` instead of `replaceFirst`. Add check on `Bearer` in `supports` method